server/php/cherrypy/src/webapp/dispatchers.py

```diff
@@ -255,6 +255,20 @@ def _validate_test_run(self, submit_id, metadata, data):
         return None
 
     #
+    # GET /submit/ : Server status
+    #
+    @cherrypy.tools.json_out()
+    def GET(self, **kwargs):
+        prefix = '[GET /submit/]'
+        self.logger.debug(prefix)
+
+        rtn = {}
+        rtn['status'] = 0
+        rtn['status_message'] = 'Success (submit)'
+
+        return rtn
```

Comment:
What's the difference between GET() on line 261 and GET() on line 275?

Reply:
Ah yes. I already brought that over. I'll push a fix today.

Reply:
fixed in latest push. Thanks